platform/graphics/layer_opengl.c

```diff
@@ -161,6 +166,90 @@ static void GLSetCullMode(PLCullMode mode) {
     gfx_state.current_cullmode = mode;
 }
 
+
+
+/////////////////////////////////////////////////////////////
+// Framebuffer
+
+static unsigned int TranslateFrameBufferBinding(PLFBOTarget targetBinding) {
+    switch(targetBinding) {
+        case PL_FRAMEBUFFER_DEFAULT:    return GL_FRAMEBUFFER;
+        case PL_FRAMEBUFFER_DRAW:       return GL_DRAW_FRAMEBUFFER;
+        case PL_FRAMEBUFFER_READ:       return GL_READ_FRAMEBUFFER;
+        default:                        return 0;
+    }
+}
+
+static void GLCreateFrameBuffer(PLFrameBuffer *buffer) {
+
+    glGenFramebuffers(1, &buffer->fbo );
+    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, buffer->fbo);
+    GfxLog("Created framebuffer %dx%d", buffer->width, buffer->height);
+
+    if(buffer->flags & PL_BUFFER_COLOUR){
+            glGenRenderbuffers(1, &buffer->rbo[0]);
```

Comment:
Rather than using an array here, can you use variable names instead, e.g. 'buffer->rbo_colour' etc. just aids readability (or use macros for each index if you feel this should remain an array).